src/Pods/Admin/Config/Pod.php

```diff
@@ -712,11 +712,12 @@ public function get_fields( \Pods\Whatsit\Pod $pod, array $tabs ) {
 					'boolean_yes_label' => '',
 				],
 				'has_archive_slug'       => [
-					'label'      => __( 'Archive Page Slug Override', 'pods' ),
-					'help'       => __( 'If archive page is enabled, you can override the slug used by WordPress, which defaults to the name of the post type.', 'pods' ),
-					'type'       => 'text',
-					'default'    => '',
-					'depends-on' => [ 'has_archive' => true ],
+					'label'         => __( 'Archive Page Slug Override', 'pods' ),
+					'help'          => __( 'If archive page is enabled, you can override the slug used by WordPress, which defaults to the name of the post type.', 'pods' ),
+					'type'          => 'slug',
+					'slug_fallback' => '-',
+					'default'       => '',
+					'depends-on'    => [ 'has_archive' => true ],
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 8171cf6054fa20ae3eb034d165f00411</sub>